packages/toolkit/src/createDraftSafeSelector.ts

```diff
@@ -8,11 +20,5 @@ import { createSelector } from 'reselect'
  * that might be possibly outdated if the draft has been modified since.
  * @public
  */
-export const createDraftSafeSelector: typeof createSelector = (
-  ...args: unknown[]
-) => {
-  const selector = (createSelector as any)(...args)
-  const wrappedSelector = (value: unknown, ...rest: unknown[]) =>
-    selector(isDraft(value) ? current(value) : value, ...rest)
-  return wrappedSelector as any
-}
+export const createDraftSafeSelector =
+  createDraftSafeSelectorCreator(defaultMemoize)
```

Comment:
this technically means it isn't using the normal `createSelector` instance from reselect - is that an issue?